library/blivet.py

```diff
@@ -19,6 +19,7 @@
 version_added: "2.5"
 
 description:
+    - WARNING: Do not use this module directly! It is only for role internal use.
```

Comment:
:-( These have to be quoted - so `- "WARNING: Do not use this module directly! It is only for role internal use."`
We'll also have to fix kernel_settings role

Reply:
+1 Correct. That's what I was about to ask you... (I noticed in network_connections and have just fixed it.)
Otherwise, it's converted like this... (;_;)
```
"description": [
      {
             "WARNING": "Do not use this module directly! It is only for role internal use."
      },
      "This module accepts various forms of block device identifiers and resolves them to the correct block device node path."
],
```